client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -22,6 +24,12 @@ const AuthMenu = (): JSX.Element => {
     handleClose();
     logout();
   };
+  const handleProfile = () => {
+    history.push('/profiledetails');
+  };
+  const handlemybookings = () => {
+    history.push('/bookings');
+  };
```

Comment:
Couldn't we do this with the Link component without having to have two functions?

Reply:
These are temporary links, I will remove them.